section/models.py

```diff
@@ -303,9 +303,12 @@ def get_context(self, request, *args, **kwargs):
         return context
     
     def get_section_articles(self, order='-explicit_published_at') -> QuerySet:
+        import datetime
         # return ArticlePage.objects.from_section(section_root=self)
         # section_articles = ArticlePage.objects.live().public().filter(current_section=self.slug).order_by(order)
-        section_articles = ArticlePage.objects.live().public().descendant_of(self).order_by(order)
+        #section_articles = ArticlePage.objects.live().public().descendant_of(self).order_by(order)
```

Comment:
To keep the code clean and avoid confusion, I'd recommend deleting these commented lines. We can always look back in the git history to see previous versions of the code